src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -127,6 +126,7 @@ object EasyIngest {
     for {
       foxml <- getFOXML(sdo)
       pid <- executeIngest(configDictionary(sdo.getName), foxml)
+      _ = log.info(s"ingested $pid $foxml")
```

Comment:
Ah, thought it was the full xml, but if it is the file name/path I'm fine with that.